### ISSUE TITLE
React 17 support

### ISSUE THREAD

**Issue by cbratschi** [JoschuaSchneider/use-error-boundary#25]
Error while installing with React 17 dev dependency:

```
npm i use-error-boundary

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: abc@1.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.13.1" from use-error-boundary@2.0.5
npm ERR! node_modules/use-error-boundary
npm ERR!   use-error-boundary@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Are there plans to support React 17?

**Comment by JoschuaSchneider** [comment]
Hey, yes this package should not break with React 17.

Until I update the peer dependency, you can install the package with either `--legacy-peer-deps` or `--force`.

**Comment by JoschuaSchneider** [comment]
I fixed this in v2.0.6 🎉 

Thanks for reporting the Issue!

**Comment by cbratschi** [comment]
@cbratschi Thanks a lot!
